### PR TITLE
pd_client: move ReadableDuration to failpoint

### PR DESCRIPTION
**Pull request by HuSharp**
Signed-off-by: husharp <jinhao.hu@pingcap.com>

### What is changed and how it works?

Issue Number: ref #13673

What's Changed:

```commit-message
remove `unused_imports` when make release.
```

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by JmPotato — APPROVED** [review]

File: components/pd_client/src/client_v2.rs [hunk 1]
```diff
@@ -71,6 +69,8 @@ use crate::PdFuture;
 fn request_timeout() -> Duration {
     fail_point!("pd_client_v2_request_timeout", |s| {
         use std::str::FromStr;
+
+        use tikv_util::config::ReadableDuration;
```
Comment (tabokie) [inline components/pd_client/src/client_v2.rs:73]:
```suggestion
        use std::str::FromStr;
        use tikv_util::config::ReadableDuration;
```

File: components/pd_client/src/client_v2.rs [hunk 2]
```diff
@@ -412,6 +412,8 @@ async fn reconnect_loop(
     let backoff = (|| {
         fail_point!("pd_client_v2_backoff", |s| {
             use std::str::FromStr;
+
+            use tikv_util::config::ReadableDuration;
```
Comment (tabokie) [inline components/pd_client/src/client_v2.rs:416]:
```suggestion
            use std::str::FromStr;
            use tikv_util::config::ReadableDuration;
```

Comment (HuSharp) [reply components/pd_client/src/client_v2.rs]: This is the "format" suggestion 😂 I don't want this either (

File: components/pd_client/src/client_v2.rs [hunk 3]
```diff
@@ -412,6 +411,7 @@ async fn reconnect_loop(
     let backoff = (|| {
         fail_point!("pd_client_v2_backoff", |s| {
             use std::str::FromStr;
+            use tikv_util::config::ReadableDuration;
```
Comment (tabokie) [inline components/pd_client/src/client_v2.rs:414]:
```suggestion
            use std::str::FromStr;

            use tikv_util::config::ReadableDuration;
```

File: components/pd_client/src/client_v2.rs [hunk 4]
```diff
@@ -71,6 +69,7 @@ use crate::PdFuture;
 fn request_timeout() -> Duration {
     fail_point!("pd_client_v2_request_timeout", |s| {
         use std::str::FromStr;
+        use tikv_util::config::ReadableDuration;
```
Comment (tabokie) [inline components/pd_client/src/client_v2.rs:72]:
```suggestion
        use std::str::FromStr;

        use tikv_util::config::ReadableDuration;
```

**Pull request merged**